core/chains/evm/logpoller/disabled.go

```diff
@@ -95,6 +95,6 @@ func (d disabled) IndexedLogsCreatedAfter(eventSig common.Hash, address common.A
 	return nil, ErrDisabled
 }
 
-func (d disabled) LatestBlockByEventSigsAddrsWithConfs(eventSigs []common.Hash, addresses []common.Address, confs int, qopts ...pg.QOpt) (int64, error) {
+func (d disabled) LatestBlockByEventSigsAddrsWithConfs(fromBlock int64, eventSigs []common.Hash, addresses []common.Address, confs int, qopts ...pg.QOpt) (int64, error) {
```

Comment:
Alright, makes sense and seems useful.  I'll have to remember to update the logpoller dev guide with the new signature.